src/engine/Service.cpp

```diff
@@ -125,6 +125,16 @@ ProtoResult Service::computeResultImpl([[maybe_unused]] bool requestLaziness) {
   ad_utility::httpUtils::Url serviceUrl{
       asStringViewUnsafe(parsedServiceClause_.serviceIri_.getContent())};
 
+  // Receive updates about the RuntimeInformation from the service endpoint.
+  // Note: Regular SPARQL endpoints do not support RTI retrieval using
+  // a websocket connection, this works for QLever endpoints only.
+  const std::string queryId = ad_utility::UuidGenerator()();
+  const std::string wsTarget = absl::StrCat(WEBSOCKET_PATH, queryId);
+  auto runtimeInfoClient = networkFunctions_.getRuntimeInfoClient_(
+      serviceUrl, wsTarget,
+      std::bind(&Service::handleChildRuntimeInfoUpdate, this,
+                std::placeholders::_1));
```

Comment:
Use `std::bind_front`, `std::bind` is outdated.
Or better use `absl::bind_front` (from `<absl/functioal/bind_front.h>` to support our backport to C++17